packages/addon-rslib/src/init.ts

```diff
@@ -0,0 +1,53 @@
+import { createRsbuild, mergeRsbuildConfig } from '@rsbuild/core'
+import { unstable_composeCreateRsbuildConfig } from '@rslib/core'
+
+import type { RsbuildConfig, RsbuildInstance } from '@rsbuild/core'
+import type { RslibConfig } from '@rslib/core'
+
+export async function startMFDevServer(
```

Comment:
```suggestion
export async function startMfDevServer(
```
as `mf` in `mfRsbuildConfig` is not capitalized.

Reply:
this either all capital letters or all small letters, so better to keep captital~